src/css/controls/imports/slider.less

```diff
@@ -185,3 +185,21 @@
         outline: @ui-focus;
     }
 }
+
+.jw-breakpoint--1 {
+    .jw-slider-time {
```

Comment:
Prior to these changes, a lot of slider-time styles also get applied to horizontal volume slider. Does the horizontal volume slider look as expected given the changes?

Reply:
ah i totally forgot about audio mode, great catch. have some updates to make there. will ping you when that's fixed

Reply:
Given the AC around this ticket specifically related to the video view of the player, i've disabled these styles for the audio player because they don't make that much sense. Along with that, the audio player is much better at the lower sizes so I don't see a need to retune that. Will clear with alicia but should be good for review now!

Reply:
Got confirmation these changes are video player only, merging!